stepup-build2.sh

```diff
@@ -14,21 +14,49 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-CWD=`pwd`
-COMPONENTS=("Stepup-Middleware" "Stepup-Gateway" "Stepup-SelfService" "Stepup-RA" "Stepup-tiqr" "Stepup-Azure-MFA" "Stepup-Webauthn")
-BUILD_ENV=prod
 
+CWD=`pwd`
+# List of supported components
+COMPONENTS=("Stepup-Middleware" "Stepup-Gateway" "Stepup-SelfService" "Stepup-RA" "Stepup-tiqr" "Stepup-Webauthn" "oath-service-php"  "Stepup-Azure-MFA")
+BUILD_ENV=build
+SYMFONY_ENV=prod # Default, set to BUILD_ENV for composer install
+
```

Comment:
`BUILD_ENV=build` while `SYMFONY_ENV=prod`. This is intentional?

Reply:
Yes I picked that up from the `master` tapes. Composer listens to symfony_env, and the rest of the console commands use the other. This worked, but using `prod` exclusively does also. Maybe it makes more sense to align them to both use prod. I will update this. 

Thanks for the review